presto-docs/src/main/sphinx/release/release-0.194.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.194
+=============
+
+General Changes
+---------------
+
+* Fix planning performance regression when querying tables with many partitions.
+* Fix deadlock in memory management logic introduced in the previous release.
+* Add :func:`word_stem` function.
```

Comment:
I remember that new features are supposed to be in the middle?

> When writing release notes, try to organize the items so that categories of changes are grouped together. It makes it easier to scan the list.
> 
> I generally prefer putting all bug fixes at the top, followed by new features, and, finally, performance optimizations.
> 
> Martin